docs(color): Add new color palette, link to old palette

glynnis opened the pull request:
https://jira.rax.io/browse/FRMW-259

New color palette, ready for review!

- [x] Dev LGTM
- [x] Dev LGTM
- [x] Design LGTM

glynnis commented:
![screen shot 2015-10-22 at 9 02 31 am](https://cloud.githubusercontent.com/assets/1529366/10667154/78d9858c-789b-11e5-9726-c7f2c668fdc6.png)
![screen shot 2015-10-22 at 9 02 48 am](https://cloud.githubusercontent.com/assets/1529366/10667153/78d9439c-789b-11e5-8bb8-dad3690cd1f3.png)
![screen shot 2015-10-22 at 9 02 41 am](https://cloud.githubusercontent.com/assets/1529366/10667155/7aa47264-789b-11e5-8a41-96a3e134a792.png)

CITguy commented:
So are we no longer displaying the color hex values?

CITguy commented:
All of the palettes use the "500" value as the background for their title. Should the gray scale do the same?

glynnis commented:
@CITguy Consulted with @ElementE, and he sees the gray scale as its own special snowflake, so calling out a primary there isn't necessary.

Hex values aren't being displayed since variables are provided via the Framework, and we want to encourage projects to use variables rather than hard-coding colors at the project level (so there's some degree of future-proofing).

glynnis commented:
Thinking more about it, and wondering if designers will still need hex values to be present :confused: 

Maybe we show both? I just want to be sure it's clear to devs that the variables are the only thing they should use in LESS, but I suppose we could catch that kind of thing with a linter?

CITguy commented:
There might be a possibility to check that colors are used only for variables. `lesshint` doesn't have support for that yet, though.

CITguy commented on src/rxApp/rxApp.less:
```diff
@@ -481,7 +481,7 @@
         & > .rx-notifications {
             position:fixed;
-            z-index: 5;
+            z-index: 20;  // accommodate rxSelect .fake-select class
```
This change was made in another PR. You might want to rebase.

glynnis commented:
Updated this branch and removed the old screen shots. Not including "before" shots since a lot of this is new content.

Added hex codes, content, and color usage guide.

![screen shot 2015-10-27 at 3 31 47 pm](https://cloud.githubusercontent.com/assets/1529366/10771663/f60f62b0-7cbf-11e5-99ab-00261e7cbed5.png)
![screen shot 2015-10-27 at 3 31 53 pm](https://cloud.githubusercontent.com/assets/1529366/10771665/f612417e-7cbf-11e5-9a01-5725282f5f93.png)
![screen shot 2015-10-27 at 3 32 19 pm](https://cloud.githubusercontent.com/assets/1529366/10771664/f610ab34-7cbf-11e5-8463-126bef7656e5.png)

glynnis commented on demo/assets/demo.less:
```diff
@@ -133,6 +133,10 @@ table.component-attributes {
     }
 }
 
+.page-content {
```
Added this in because of some issues that came in from `master` with titles and floats.

glynnis commented on demo/styles/buttons.html:
```diff
@@ -1,4 +1,4 @@
-<rx-page title="'Buttons &amp; Links'" class="stylguide">
+<rx-page title="'Buttons &amp; Links'" class="styleguide">
```
Unrelated change -- found a typo. Sneaking it in with this PR.

glynnis commented on demo/styles/color-1-0.html:
```diff
@@ -0,0 +1,104 @@
+<rx-page title="'Color'" class="styleguide">
```
Should I change this title to something like "1.0 Color Palette (Deprecated)"?

ssalinas24 replied:
@glynnis I would agree with that.

glynnis commented:
Removing "DO NOT MERGE" as this is ready for review.

glynnis commented:
Screenshots updated to reflect that I merged in @CITguy's branch.

CITguy commented:
@glynnis please rebase

CITguy commented on demo/styles/color.html:
```diff
@@ -1,101 +1,322 @@
+    available at the project level. Using them ensures consistency across Encore
+    as colors shift or change over time, and will save you from having to revise
+    hard-coded hex codes in your code the future. Hex codes are provided here
+    for reference only.
```
Missing closing `</p>`

CITguy commented on src/rxApp/common.less:
```diff
@@ -2,6 +2,7 @@
  * Common
  */
 @import url("https://fonts.googleapis.com/css?family=Roboto:400,100,100italic,300,300italic,400italic,700,700italic");
+@import url("https://fonts.googleapis.com/css?family=Roboto+Mono:400,100,100italic,300,300italic,400italic,700,700italic");
```
Yay! "Roboto Mono"

ssalinas24 commented on demo/styles/color.html:
```diff
@@ -1,101 +1,322 @@
+    <strong>only use the variable names provided, NOT hex codes</strong>.
+    The LESS variables for colors ship by default with the Framework and are
+    available at the project level. Using them ensures consistency across Encore
+    as colors shift or change over time, and will save you from having to revise
```
@glynnis "revise hard-coded hex codes in your code the future."
could probably change to:
"revise hard-coded hex codes in the future."

ssalinas24 commented on demo/styles/color.html:
```diff
@@ -1,101 +1,322 @@
-          <li>
-            Should not be used to communicated "active" or "success"
-            (since those are states, not actions a user can take)
+            Should not be used to communicated "active" or "success" (since those are
```
@glynnis "communicated" should be "communicate".

ssalinas24 commented on demo/styles/color.html:
```diff
@@ -1,101 +1,322 @@
-            Should not be used to communicated "active" or "success"
-            (since those are states, not actions a user can take)
+            Should not be used to communicated "active" or "success" (since those are
+            states, not actions a user can take
```
@glynnis Also need a ")" at the end of the line.

ssalinas24 commented on demo/styles/color.html:
```diff
@@ -1,101 +1,322 @@
+      <div class="detail">
+        <ul class="list">
+          <li>
+            Used to communicated a button, link, or element is disabled.
```
@glynnis "communicated" should be "communicate" here also.

ssalinas24 commented on demo/styles/color-1-0.html:
```diff
@@ -0,0 +1,104 @@
+            actions.
+          </li>
+          <li>
+            Should not be used to communicated "active" or "success"
```
@glynnis "communicated" should be "communicate"

ssalinas24 commented on demo/styles/color-1-0.html:
```diff
@@ -0,0 +1,104 @@
+        #dededd - @disabled-gray
+      </div>
+      <div class="swatch-body">
+        Used to communicated a button, link, or element is disabled.
```
"communicated" should be "communicate that" here as well.

glynnis commented:
Responded to feedback and rebased/squashed.

glynnis commented:
More :eyes: :eyes: :eyes:  plz

CITguy commented:
LGTM

ssalinas24 commented:
LGTM

bry4n commented:
Woot Color 2.0!

LGTM
